computer_vision/README.md

```diff
@@ -1,7 +1,11 @@
-### Computer Vision
+# Computer Vision
+
+Computer vision is a field of computer science that works on enabling computers to see, identify and process images in the same way that human vision does, and then provide appropriate output.
```

Comment:
or just understand

Reply:
I guess you could argue that computer "understand" what they are seeing but I think it's clearer as it is right now. We can change it if you think it's not